### ISSUE TITLE
CORS Error

### ISSUE THREAD

**Issue by papuruth** [FaridSafi/react-native-google-places-autocomplete#644]
# Describe the bug

Access to XMLHttpRequest at 'https://maps.googleapis.com/maps/api/place/autocomplete/json?&input=Lan&key=""&type=address&origin=&language=en' from origin 'http://localhost:8081' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

The app is running on a physical device in dev mode.

<details>
  <summary>Click to expand!</summary>
  
  ```javascript
    <GooglePlacesAutocomplete
            placeholder="Search"
            onPress={(data, details = null) => {
              // 'details' is provided when fetchDetails = true
              console.log(data, details);
            }}
            returnKeyType="Search"
            fetchDetails
            GooglePlacesSearchQuery={{
              rankby: 'distance'
            }}
            GooglePlacesDetailsQuery={{
              fields: 'formatted_address'
            }}
            query={{
              key: 'removed for security purpose',
              type:'address',
              origin: null,
              language: 'en',
            }}
            currentLocation
            currentLocationLabel="Current location"
          />    
  ```
</details>

# Additional context

- Library Version: 2.0.3
- React Native Version: 0.63.2

- [ ] iOS
- [x] Android
- [ ] Web

Add any other context about the problem here, screenshots

**Comment by bell-steven** [comment]
Are you using a snack and running it on your device by any chance?  I can't reproduce this on a device.  I believe only a browser has CORS.

**Comment by papuruth** [comment]
No, I'm using RN CLI and the app is running on a physical device

**Comment by bell-steven** [comment]
I cannot reproduce this.  The only think that I can think of is leaving out `origin: null`.

**Comment by bell-steven** [comment]
Closing, as I cannot reproduce this. If anybody else comes across this problem, please open a new issue.

**Comment by ravik2015** [comment]
Please re-open facing same issue

**Comment by ravik2015** [comment]
> # Describe the bug
> Access to XMLHttpRequest at '[https://maps.googleapis.com/maps/api/place/autocomplete/json?&input=Lan&key=""&type=address&origin=&language=en](https://maps.googleapis.com/maps/api/place/autocomplete/json?&input=Lan&key=%22%22&type=address&origin=&language=en)' from origin 'http://localhost:8081' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
> 
> The app is running on a physical device in dev mode.
> 
> Click to expand!
> ```js
>   <GooglePlacesAutocomplete
>           placeholder="Search"
>           onPress={(data, details = null) => {
>             // 'details' is provided when fetchDetails = true
>             console.log(data, details);
>           }}
>           returnKeyType="Search"
>           fetchDetails
>           GooglePlacesSearchQuery={{
>             rankby: 'distance'
>           }}
>           GooglePlacesDetailsQuery={{
>             fields: 'formatted_address'
>           }}
>           query={{
>             key: 'removed for security purpose',
>             type:'address',
>             origin: null,
>             language: 'en',
>           }}
>           currentLocation
>           currentLocationLabel="Current location"
>         />    
> ```
> 
> # Additional context
> * Library Version: 2.0.3
> * React Native Version: 0.63.2
> * [ ]  iOS
> * [x]  Android
> * [ ]  Web
> 
> Add any other context about the problem here, screenshots

@papuruth did you find any solution ??
For me its happening only if debugging is enabled

**Comment by bell-steven** [comment]
Please provide a **reproducible** example.  
If I cannot reproduce the issue, I am shooting in the dark.

**Comment by erikadelao13** [comment]
This same issue is happening to me, only if debugging is enabled, otherwise, it works ok.

**Comment by sandeepvishnu** [comment]
I am facing same issue, does anyone knows the solution for this. Thanks

**Comment by robbporto** [comment]
The same problem is happening to me to.

**Comment by Srezzx** [comment]
Hey, 
What @erikadelao13 said worked for me. This issue only occurs when react-native-debugger is on. Worked smoothly after it was turned off.

ENV - 
 Windows 
 Android Simulator
 React-native  : 0.70.3
